### PR TITLE
Please review this, @casey-collab

### PR DESCRIPTION
**Pull request by Wenhua-Xue**
Add some funny stuff into Chinese story.

[Review on Reviewable](https://reviewable.io/reviews/udacity/create-your-own-adventure/2072)

### CODE REVIEW

**Comment by cbuckey-uda** [comment]
@casey-collab, please review this

File: Chinese/wx/SnowWhite.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+话说孙悟空到了西域后，见到了前来晚宴的恶毒皇后以及她的继女
```
Comment (casey-collab) [inline Chinese/wx/SnowWhite.md]: It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by cbuckey-uda** [comment]
Closing this pull request due to lack of activity. If you ever have time to respond to the comments above, feel free to re-open it.

**Pull request closed without merging**